functions/notify_slack.py

```diff
@@ -16,6 +16,17 @@ def decrypt(encrypted_url):
     except Exception:
         logging.exception("Failed to decrypt URL with KMS")
 
+def asg_notification(message, regions):
+    return {
+        "color": 'good',
```

Comment:
at some point we may want to add better support for the sad path and make this `danger` color for those. https://docs.aws.amazon.com/autoscaling/ec2/userguide/cloud-watch-events.html#launch-unsuccessful has some examples which have:
```
  "detail": {
      "StatusCode": "Failed",
``` 
which might be worth checking this attribute for `Failed` and go with `danger` otherwise `good`.

Reply:
Added functionality for this.

Test example:
https://smartrentcom.slack.com/archives/CK5A63GUU/p1624552878141700